lib/plugins/aws/invokeLocal/index.js

```diff
@@ -215,47 +226,28 @@ class AwsInvokeLocal {
   }
 
   checkDockerDaemonStatus() {
-    return new BbPromise((resolve, reject) => {
-      const docker = spawn('docker', ['version']);
-      docker.on('exit', error => {
-        if (error) {
-          reject('Please start the Docker daemon to use the invoke local Docker integration.');
-        }
-        resolve();
-      });
+    return spawnExt('docker', ['version']).catch(() => {
+      throw Error('Please start the Docker daemon to use the invoke local Docker integration.');
```

Comment:
`throw new Error` (technically linter should pick this, I'll ensure it's fixed on it's side)

Reply:
Seems not adddressed